airflow/configuration.py

```diff
@@ -1545,6 +1548,12 @@ def initialize_config() -> AirflowConfigParser:
     return local_conf
 
 
+def make_group_other_inaccessible(file_path: str):
+    with suppress(Exception):
+        permissions = os.stat(file_path)
+        os.chmod(file_path, permissions.st_mode & (stat.S_IRUSR | stat.S_IWUSR))
```

Comment:
Should we raise an error instead of suppress it?

Reply:
I'd say log an error, not to raise it. There might be various reasons why changing permissions is not possible - for example when the filesystem config file gets generated on happens to be NFS.  Failing hard in this case is kinda strange at this point as the file has already been generated (so next time if we run airflow it will succeed because the whole branch is skiped in this case.

Reply:
Changed.